### ISSUE TITLE
Don't Remove JSON+LD

### ISSUE THREAD

**Issue by youmustfight** [jafaircl/gatsby-plugin-amp#18]
In the process of pruning out javascript from the header, it'd be great to keep the JSONLD script tag:
```
<script type="application/ld+json">
[{
  "@context":"http://schema.org",
  "@type":"BreadcrumbList",
  "itemListElement": [],
  ...
}]
</script>
```

[It's used by crawlers](https://developers.google.com/search/docs/guides/intro-structured-data) to easily get structured data about the page and as a result those pages get special treatment in search results (via "rich snippets")

Since switching over to `gatsby-plugin-amp`, I noticed the application/ld+json tag is getting pruned. I'm no longer getting a "This page has valid structured data" message from Google's AMP tester.

Here are results from Google's search console [AMP testing tool](https://search.google.com/test/amp)

### Production (using my hacky regex replacement script)
![image](https://user-images.githubusercontent.com/4956240/57532893-86cd9d00-730a-11e9-8b6a-432a9db573d2.png)

### Staging (using gatsby-plugin-amp)
![image](https://user-images.githubusercontent.com/4956240/57532874-787f8100-730a-11e9-9a51-340d7dd1ed64.png)

**Comment by jafaircl** [comment]
Sorry it's taken so long to reply. I'll take a look and see if there's a way to not remove json+ld scripts

**Comment by jafaircl** [comment]
Should be fixed in 0.2.2

**Comment by youmustfight** [comment]
Thanks @jafaircl ! 😄

**Comment by nyedidikeke** [comment]
Awesome 👍 

> Should be fixed in 0.2.2

When is it due @jafaircl?
